docs/kubernetes/how-to-migrate-from-k8s-alpha-to-terraform/index.md

```diff
@@ -0,0 +1,257 @@
+
+1.  To scale your cluster, edit `cluster.tf` with the text editor of your choice. The contents will be similar to the following:
+
+    {{< file "cluster.tf" >}}
+variable "server_type_node" {
+  default = "g6-standard-2"
+}
+variable "nodes" {
+  default = 3
+}
+variable "server_type_master" {
+  default = "g6-standard-2"
+}
+variable "region" {
+  default = "us-east"
+}
+variable "ssh_public_key" {
+  default = "/Users/username/.ssh/id_rsa.pub"
+}
+module "k8s" {
+  source  = "git::https://github.com/linode/terraform-linode-k8s.git?ref=for-cli"
+
+  linode_token = "<your api token>"
+
+  linode_group = "kabZmZ3TA0r-mycluster"
+
+  server_type_node = "${var.server_type_node}"
+
+  nodes = "${var.nodes}"
+
+  server_type_master = "${var.server_type_master}"
+
+  region = "${var.region}"
+
+  ssh_public_key = "${var.ssh_public_key}"
+}
+{{< /file >}}
+
+1.  To scale your cluster, edit the value of the `nodes` variable. To resize the number of nodes from `3` to `5`, make the following edit and save your changes:
+
+        variable "nodes" {
+          default = 5
+        }
+
+1.  Once your edit is made, move back to the `/.k8s-alpha-linode/clustername` directory and apply your changes with Terraform:
+
+        terraform apply
+
+    {{< note >}}
+You may need to use the original Terraform version used to deploy the cluster (either Terraform 0.11.X or Terraform 0.12.X). If you do not, you will see syntax errors.
+{{< /note >}}
+
+1.  Once this is completed, you'll see a prompt reviewing your changes and asking if you would like to accept them. To proceed, type `yes` and Terraform will proceed to make the changes. This process may take a few moments.
+
+    {{< note >}}
+When prompted, you may notice that one item in your plan is marked as `destroy`. This is generally a "null resource" or a local script execution and is not indicative of an unintended change.
+{{</ note >}}
+
+1.  After Terraform has finished, your cluster will be resized. To confirm, enter the following command to list all nodes in your cluster, replacing the string `mycluster` with the name of the cluster you edited:
+
+        kubectl --kubeconfig=mycluster.conf get nodes
+
+1.  The output will then list an entry for each node:
+
+    {{< output >}}
+kubectl --kubeconfig=mycluster.conf get nodes
+NAME             	STATUS   ROLES	  AGE 	  VERSION
+mycluster-master-1      Ready	 master   21m 	  v1.13.6
+mycluster-node-1 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-2 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-3 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-4 	Ready	 &lt;none&gt;   4m26s   v1.13.6
+mycluster-node-5 	Ready	 &lt;none&gt;   4m52s   v1.13.6
+{{< /output >}}
+
+### Upgrade a Cluster
+
+You may have noticed that the Terraform module, `cluster.tf`, refers to a specific branch or git commit hash referencing a remote Terraform module on GitHub. The following section will outline how to upgrade your cluster to the latest version of this module.
```

Comment:
Do we know what this referenced module does? Just wondering if it's important to explain it to a user.

Reply:
ok I changed this and updated the first paragraph a little too to be more clear about what this actually is.